x/group/server/testsuite/genesis.go

```diff
@@ -0,0 +1,172 @@
+		})
+		s.Require().NoError(err)
+		s.assertGroupAccountsEqual(g, res.Info)
+	}
+
+	for _, g := range genesisState.Proposals {
+		res, err := s.queryClient.Proposal(ctx, &group.QueryProposalRequest{
+			ProposalId: g.ProposalId,
+		})
+		s.Require().NoError(err)
+		s.assertProposalsEqual(g, res.Proposal)
+
+		votesRes, err := s.queryClient.VotesByProposal(ctx, &group.QueryVotesByProposalRequest{
+			ProposalId: g.ProposalId,
+		})
+		s.Require().NoError(err)
+		s.Require().Equal(len(votesRes.Votes), 1)
+		s.Require().Equal(votesRes.Votes[0], genesisState.Votes[0])
+	}
+
+	exported, err := s.exportGenesisHandler(ctx, cdc)
+	s.Require().NoError(err)
+
+	var exportedGenesisState group.GenesisState
+	err = cdc.UnmarshalJSON(exported, &exportedGenesisState)
+	s.Require().NoError(err)
+
+	s.Require().Equal(genesisState.Groups, exportedGenesisState.Groups)
+	s.Require().Equal(genesisState.GroupMembers, exportedGenesisState.GroupMembers)
+
+	s.Require().Equal(len(genesisState.GroupAccounts), len(exportedGenesisState.GroupAccounts))
+	for i, g := range genesisState.GroupAccounts {
+		res := exportedGenesisState.GroupAccounts[i]
+		s.Require().NoError(err)
+		s.assertGroupAccountsEqual(g, res)
+	}
+
+	s.Require().Equal(len(genesisState.Proposals), len(exportedGenesisState.Proposals))
+	for i, g := range genesisState.Proposals {
+		res := exportedGenesisState.Proposals[i]
+		s.Require().NoError(err)
+		s.assertProposalsEqual(g, res)
+	}
+	s.Require().Equal(genesisState.Votes, exportedGenesisState.Votes)
+
+	s.Require().Equal(genesisState.GroupSeq, exportedGenesisState.GroupSeq)
+	s.Require().Equal(genesisState.GroupAccountSeq, exportedGenesisState.GroupAccountSeq)
+	s.Require().Equal(genesisState.ProposalSeq, exportedGenesisState.ProposalSeq)
+
+}
+
+func (s *IntegrationTestSuite) assertGroupAccountsEqual(g *group.GroupAccountInfo, other *group.GroupAccountInfo) {
+	s.Require().Equal(g.GroupAccount, other.GroupAccount)
+	s.Require().Equal(g.GroupId, other.GroupId)
+	s.Require().Equal(g.Admin, other.Admin)
+	s.Require().Equal(g.Metadata, other.Metadata)
+	s.Require().Equal(g.Version, other.Version)
+	s.Require().Equal(g.GetDecisionPolicy(), other.GetDecisionPolicy())
+}
+
+func (s *IntegrationTestSuite) assertProposalsEqual(g *group.Proposal, other *group.Proposal) {
+	s.Require().Equal(g.ProposalId, other.ProposalId)
+	s.Require().Equal(g.GroupAccount, other.GroupAccount)
+	s.Require().Equal(g.Metadata, other.Metadata)
+	s.Require().Equal(g.Proposers, other.Proposers)
+	s.Require().Equal(g.SubmittedAt, other.SubmittedAt)
+	s.Require().Equal(g.GroupVersion, other.GroupVersion)
+	s.Require().Equal(g.GroupAccountVersion, other.GroupAccountVersion)
+	s.Require().Equal(g.Status, other.Status)
+	s.Require().Equal(g.Result, other.Result)
+	s.Require().Equal(g.VoteState, other.VoteState)
+	s.Require().Equal(g.Timeout, other.Timeout)
+	s.Require().Equal(g.ExecutorResult, other.ExecutorResult)
+	s.Require().Equal(g.GetMsgs(), other.GetMsgs())
```

Comment:
in new tests, how about using `
```
require := s.Require()
...
require.Equal(...)
```
It makes the code easier to read (less parenthesis) and a bit shorter .